pkg/fakerp/config.go

```diff
@@ -34,6 +41,7 @@ type Config struct {
 	NoGroupTags      bool   `envconfig:"NOGROUPTAGS"`
 	ResourceGroupTTL string `envconfig:"RESOURCEGROUP_TTL"`
 	Manifest         string `envconfig:"MANIFEST"`
+	SecretsDir       string `envconfig:"SECRETSDIR"`
```

Comment:
can we avoid adding this?

Reply:
we kinda do. Totally our CI limitation. We have our CI in node build do `git clone openshift-azure` and default secret path is under this cloned directory tree. So when injecting via secret mount we need to mount it in the other place. 
We could do some copy/linking in the CI, but we already have way too much complexity in our CI :/

Reply:
I would also remove this if the issue is just the node build job.

Reply:
For now yes. Idea was that all our secrets are under ` source /etc/azure/credentials/` in CI, so we dont need to change a lot and just point to this location. Otherwise, we need either create soft-link or copy artefacts to `openshift-azure/secrets` before creating the cluster.